dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+_Stake_ is the literal amount of BSQ specified for the vote transaction. This is BSQ you own that you would like to devote to your vote. The more BSQ you allot, the more weight your vote will hold. BSQ specified for stake is unspendable for the duration of the voting cycle.
+
+**Total voting weight is the sum of reputation value and specified stake.**
+
+Reputation is worth more than stake. The idea is that there is no better measure of good-will and dedication to the project than actually working on the project. The arrangement still allows people who buy into the project considerable influence, while hopefully minimizing the potential of whales with bad intentions from buying their way in and sabotaging the project.
+
+===== Other Details
+
+Votes are published on the Bitcoin network and _encrypted_ vote data objects are broadcast on the Bisq P2P network (hence the term 'blind' vote). The Bitcoin transaction includes a hash of the vote data object. BSQ specified for voting stake is also specified in this transaction, and cannot be spent until the vote reveal transaction is published.
+
+Following the end of the blind voting phase, there is a break of 10 blocks to protect against reorgs.
+
+==== Vote Reveal Phase
+
+_450 blocks, about 3 days_
+
+No action is required of the user during this phase aside from having the Bisq software online at some point during the phase so the vote reveal phase so it can publish the vote reveal transaction.
+
+You should see a pop-up in the UI when the transaction has been published.
+
+===== Other Details
+
+Only a Bitcoin transaction is made in this phase—no additional data is broadcast to the Bisq P2P network.
+
+This transaction includes 2 items:
+
+1. the key to decrypt the votes cast in the blind voting phase
+2. a hash of the list of voting data objects received by the Bisq node making the transaction (ordered by transaction ID; details <<#determining-consensus,below>>)
+
+One of the inputs to this transaction is the BSQ reserved for stake, so once this transaction is published, that BSQ becomes spendable again.
+
+If a user fails to make their vote reveal transaction, their vote is rendered invalid.
+
+Again, for protection against reorgs, there is a break of 10 blocks after this phase is over.
+
+==== Vote Result Phase
+
+_10 blocks, about 100 minutes_
+
+This phase is more of a trigger, so allotting 10 blocks is more for consistency than a hard requirement.
+
+In this phase, Bisq nodes calculate vote results from proposals, voting stake, and votes (which can now be decrypted from the vote reveal transactions).
+
+===== Determining Consensus
+
+We can trust that all Bisq nodes will get the same data from the the Bitcoin blockchain. However, we cannot be sure that all Bisq nodes have obtained identical data from the Bisq P2P network. It is critical that any potential inconsistencies be handled before determining vote results, otherwise, vote results may differ.
```

Comment:
"obtained identical data" -> it only about if u have received all data. the data need to be indentical as hash is in tx.